cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -160,23 +160,31 @@ func importPoolBuilder(cStorPool *apis.CStorPool, importOptions *ImportOptions)
 	return importAttr
 }
 
-// GetDevPath gets the path from given deviceID
-func GetDevPath(devid string) string {
-	lastindex := strings.LastIndexByte(devid, '/')
+// GetDevPathIfNotSlashDev gets the path from given deviceID if its not prefix
+// to "/dev"
+func GetDevPathIfNotSlashDev(devID string) string {
+	if len(devID) == 0 {
+		return ""
+	}
+
+	if strings.HasPrefix(devID, "/dev") {
+		return ""
+	}
+	lastindex := strings.LastIndexByte(devID, '/')
 	if lastindex == -1 {
-		lastindex = len(devid)
+		lastindex = len(devID)
```

Comment:
Without `/` means only disk name, like `sda/b/c`. In that case, it should return `empty`. Though this case is not possible, we should return empty here IMO.

Reply:
Done